waspc/src/Wasp/Util/Json.hs

```diff
@@ -16,3 +25,12 @@ parseJsonWithComments jsonStr = do
   case exitCode of
     ExitSuccess -> return $ decodeFromString response
     _exitFailure -> return $ Left stderr
+
+updateJsonFile :: (Value -> Value) -> Path' Abs (File a) -> IO (Either String ())
+updateJsonFile updateFn jsonFilePath = runExceptT $ do
+  jsonContent <- ExceptT $ eitherDecode <$> IOUtil.readFileBytes jsonFilePath
+  let updatedContent = updateFn jsonContent
+  liftIO $ writeJsonValue jsonFilePath updatedContent
```

Comment:
```suggestion
  liftIO $ writeJsonValue jsonFilePath $ updateFn jsonContent
```

Reply:
Keep it as you like, this was just an idea. I like this better because it just flows, the operations, from right to left, in one line. While in two lines it is broken a bit. But really this super silly nitpicking, I am aware, so whatever you like better.